src/test/groovy/com/bmuschko/gradle/docker/internal/RegistryAuthLocatorTest.groovy

```diff
@@ -80,16 +80,18 @@ class RegistryAuthLocatorTest extends Specification {
         0 * logger.error(*_)
     }
 
-    def "AuthLocator returns default config for Docker Desktop config without existing credentials"() {
+    def "AuthLocator works for Docker Desktop config without existing credentials"() {
```

Comment:
We should have both test cases:

1. The existing one: Config file can be read but the credentials couldn't be resolved
2. The config file can be read and the credentials as well.

Would you mind changing the code.

Reply:
@bmuschko done